fuel-tx/src/transaction/consensus_parameters.rs

```diff
@@ -5,15 +5,197 @@ use fuel_types::{
     ChainId,
 };
 
+pub mod gas;
+
+pub use gas::{
+    DependentCost,
+    GasCosts,
+    GasCostsValues,
+    GasUnit,
+};
+
 const MAX_GAS: u64 = 100_000_000;
 
+/// A collection of parameters for convenience
+#[derive(Debug, Clone)]
+pub struct ConsensusParams {
+    // TODO: This should be pub(crate) for the `fuel_tx` tests, but for some reason the
+    // tests can't see the fields with `pub(crate)`
```

Comment:
Integration tests cannot see `pub(crate)` items, as they are outside the crate. Typically you'd do a public re-export behind `test-helpers` feature flag, but I don't see much point in not having this as a part of the public interface anyways.

Reply:
Yeah. That's a good point. There are probably places outside of `fuel-tx` that could use this as well. I was referring to something in `fuel-tx/src/tests`, not in the integ tests.

I usually prefer getters/setters over accessing fields directly, but I'm not going to worry about it right now.